prototype_source/ios_gpu_workflow.rst

```diff
@@ -0,0 +1,102 @@
+(Prototype) Use iOS GPU in PyTorch
+==================================
+
+**Author**: `Tao Xu <https://github.com/xta0>`_
+
+Introduction
+------------
+
+This tutorial introduces the steps to run your models on iOS GPU. We'll be using the mobilenetv2 model as an example. Since the mobile GPU features are currently in the prototype stage, you'll need to build a custom pytorch binary from source. For the time being, only a limited number of operators are supported, and certain client side APIs are subject to change in the future versions.
+
+Mobile Preparation
+-------------------
+
+Since GPUs consume weights in a different order, the first step we have to do is to convert our torchscript mobile to a GPU compatible model. This step is also known as "prepacking". To do that, we'll build a custom pytorch binary from source that includes the Metal backend. Go ahead checkout the pytorch source code from github and run the command below
+
+.. code:: shell
+
+    cd PYTORCH_ROOT
+    USE_PYTORCH_METAL=ON python setup.py install --cmake
+
+The command above will build a custom pytorch binary from master. The ``install`` argument simply tells ``setup.py`` to override the existing PyTorch on your desktop. Once the build finished, I would recommend opening another terminal and checking the PyTorch version to see if the installation successful. As the time of writing of this recipe, the version is ``1.8.0a0+41237a4``. The number might be different depending on when you check out the code from master, but it should be greater than 1.7.0.
+
+.. code:: python
+
+    import torch
+    torch.__version__ #1.8.0a0+41237a4
+
+
+The next step is going to be converting the mobilenetv2 torchscript model to a Metal compatible model. We'll be leveraging the ``optimize_for_mobile`` API from ``torch.utils`` as shown below
+
+.. code:: python
+
+    import torch
+    from torch.utils.mobile_optimizer import optimize_for_mobile
+
+    scripted_model = torch.jit.load('./mobilenetv2.pt')
```

Comment:
replace this line with:
```
import torchvision

model_quantized = torchvision.models.quantization.mobilenet_v2(pretrained=True, quantize=True)
scripted_model = torch.jit.script(model_quantized)
```

or the non-quantized version:
```
import torchvision

model = torchvision.models.mobilenet_v2(pretrained=True)
scripted_model = torch.jit.script(model)
```

and the user can then copy and paste the while code snippet and run without getting an error "mobilenetv2.pt does not exist" and etc..